common/app/com/commercetools/sunrise/common/template/engine/handlebars/HandlebarsHelperUtils.java

```diff
@@ -8,17 +8,19 @@
 import java.util.Locale;
 import java.util.Optional;
 
+import static com.commercetools.sunrise.common.template.engine.handlebars.HandlebarsContextFactory.CMS_PAGE_IN_CONTEXT_KEY;
```

Comment:
Right now it is something shared between the two classes as I recall, both use it exactly once. I can try to review it anyway, maybe I can find a better overall structure now :)